src/backend/base/langflow/memory.py

```diff
@@ -255,7 +257,12 @@ async def astore_message(
         msg = "All of session_id, sender, and sender_name must be provided."
         raise ValueError(msg)
     if hasattr(message, "id") and message.id:
-        return await aupdate_messages([message])
+        # if message has an id and exist in the database, update it
+        # if not raise an error and add the message to the database
+        try:
+            return await aupdate_messages([message])
+        except ValueError as e:
+            logger.exception(e)
```

Comment:
You should avoid using `logger.exception`.
```suggestion
            logger.error(e)
```